<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatingArea</class>
 <widget class="QWidget" name="OperatingArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="stopMovingBtn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>255</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 27, 37);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backwardBtn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>145</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 27, 37);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play8stopBtn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>451</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);

color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>摄像头开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up8downBtn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>352</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(2, 160, 229);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>毛刷升</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fanCtrlBtn">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>34</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 201, 13);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>推进器开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GPSBtn">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>34</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 138, 138);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>GPS</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>62</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="ch1">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {   width: 100%;
height: 100%;
 border: none;
background: transparent;}
    QCheckBox {background-color: rgb(238, 27, 37);
}
</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch2">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {   width: 100%;
height: 100%;
 border: none;
background: transparent;}
    QCheckBox {background-color: rgb(238, 27, 37);
}
</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch3">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {   width: 100%;
height: 100%;
 border: none;
background: transparent;}
    QCheckBox {background-color: rgb(238, 27, 37);
}
</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch4">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {   width: 100%;
height: 100%;
 border: none;
background: transparent;}
    QCheckBox {background-color: rgb(238, 27, 37);
}
</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch5">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {   width: 100%;
height: 100%;
 border: none;
background: transparent;}
    QCheckBox {background-color: rgb(238, 27, 37);
}
</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch6">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {   width: 100%;
height: 100%;
 border: none;
background: transparent;}
    QCheckBox {background-color: rgb(238, 27, 37);
}
</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch7">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {   width: 100%;
height: 100%;
 border: none;
background: transparent;}
    QCheckBox {background-color: rgb(238, 27, 37);
}
</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch8">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {   width: 100%;
height: 100%;
 border: none;
background: transparent;}
    QCheckBox {background-color: rgb(238, 27, 37);
}
</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>82</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="switchMotorModeBtn">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 57, 60, 60);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>切换</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="forwardBtn">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 27, 37);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>前进</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="openSelectedBtn">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>179</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 201, 13);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>打开选中</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeSelectedBtn">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>288</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 201, 13);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>关闭选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open8closeBtn">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>397</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 201, 13);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>全同步</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
